src/main/java/com/welcome/bot/domain/User.java

```diff
@@ -21,42 +22,48 @@
     private String name;
 
     @Email
-    @Column(nullable = false,length = 10485760)
-    @Size(min = 2, max = 10485760)
+    @Column(nullable = false,length = 200)
+    @Size(min = 2, max = 200)
```

Comment:
Size is not right validation for this property. Better would be to use:
`@Email `
annotation.

Reply:
Resolved.